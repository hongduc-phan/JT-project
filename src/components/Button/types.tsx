export enum ButtonVariants {
  Primary = 'primary',
  Secondary = 'secondary',
  Icon = 'icon',
}
