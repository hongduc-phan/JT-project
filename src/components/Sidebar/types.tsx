export enum SidebarMenuVariant {
  Default = 'default',
  Sub = 'sub',
}
