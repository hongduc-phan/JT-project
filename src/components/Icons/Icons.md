```js
const {
  Edit,
  Delete,
  Setting,
  EPayRoll,
  ETime,
  ArrowDown,
  ArrowLeft,
  ArrowRight,
  ArrowUp,
  Notification,
  Account,
  Add,
  Search,
  Error,
} = require('./');

<div className="icon-list">
  <div>
    <div>
      <Edit />
    </div>
    <div>
      <Delete />
    </div>
    <div>
      <Setting />
    </div>
    <div>
      <Setting active={true} />
    </div>
    <div>
      <EPayRoll />
    </div>
    <div>
      <EPayRoll active={true} />
    </div>
    <div>
      <ETime />
    </div>
    <div>
      <ETime active={true} />
    </div>
    <div>
      <ArrowDown />
    </div>
    <div>
      <ArrowLeft />
    </div>
    <div>
      <ArrowUp />
    </div>
    <div>
      <ArrowRight />
    </div>
  </div>
  <div>
    <div>
      <Notification />
    </div>
    <div>
      <Account />
    </div>
    <div>
      <Add />
    </div>
    <div>
      <Search />
    </div>
    <div>
      <Error />
    </div>
  </div>
</div>;
```
