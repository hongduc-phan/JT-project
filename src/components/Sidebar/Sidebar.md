```js
const Sidebar = require('./').default;
const {ETime, EPayRoll, Setting} = require('../Icons');
const {
  SidebarButton,
  SidebarMenu,
  SidebarTitle,
  SidebarMenuItem,
  SidebarLogo,
} = require('./');

<div className="sidebar-demo">
  <Sidebar
    isExpanded={true}
    classNames={{wrap: 'sidebar-demo-in'}}
    renderExpand={
      <div>
        <SidebarTitle>Settings</SidebarTitle>
        <SidebarMenu>
          <SidebarMenuItem
            active={true}
            sub={
              <SidebarMenu variant="sub">
                <SidebarMenuItem active={true}>Company</SidebarMenuItem>
                <SidebarMenuItem>Departments</SidebarMenuItem>
                <SidebarMenuItem>Gradings</SidebarMenuItem>
                <SidebarMenuItem>Branches</SidebarMenuItem>
                <SidebarMenuItem>WorkFlows</SidebarMenuItem>
              </SidebarMenu>
            }
          >
            General
          </SidebarMenuItem>
          <SidebarMenuItem sub={<div />}>eTime</SidebarMenuItem>
          <SidebarMenuItem sub={<div />}>ePayroll</SidebarMenuItem>
          <SidebarMenuItem>Users</SidebarMenuItem>
        </SidebarMenu>
      </div>
    }
  >
    <SidebarButton>
      <ETime />
    </SidebarButton>
    <SidebarButton>
      <EPayRoll />
    </SidebarButton>
    <SidebarButton active={true}>
      <Setting active={true} />
    </SidebarButton>
    <SidebarLogo
      src="https://scontent.fsgn2-4.fna.fbcdn.net/v/t31.0-8/13268309_1776320839250890_6336739617756356471_o.jpg?_nc_cat=101&_nc_oc=AQlsWpOIc3wxFfO8CtLYt4eMBdjh7agqxOzqJNrwFOICR841FqfZjGfDqN3pzO4Y1aI&_nc_ht=scontent.fsgn2-4.fna&oh=4ca766c851f17dfccd14eac63e1bf087&oe=5D1D2F06"
      alt="JuzTalent"
    />
  </Sidebar>
</div>;
```
