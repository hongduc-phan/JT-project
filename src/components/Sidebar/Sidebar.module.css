.wrap {
  position: fixed;
  z-index: var(--z-index-sidebar);
  top: 0;
  left: 0;
  flex: 1 0 auto;
  height: 100%;
  display: flex;
}

.content {
  width: 3.5rem;
  background-color: var(--blue-dark);
  top: 0;
  left: 0;
  flex: 1 0 auto;
  height: 100%;
  display: flex;
  outline: none;
  position: absolute;
  overflow-y: auto;
  flex-direction: column;
  z-index: 2;
}

.expand {
  position: absolute;
  top: 0;
  left: -300px;
  background-color: var(--sidebar-active);
  flex: 1 0 auto;
  height: 100%;
  display: flex;
  outline: none;
  overflow-y: auto;
  flex-direction: column;
  min-width: 13.8125rem;
  z-index: 1;
  transition: left 200ms var(--decelerate-easing);
}

.expandActive {
  left: 3.5rem;
}
