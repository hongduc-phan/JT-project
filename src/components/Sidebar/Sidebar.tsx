import React, {FunctionComponent, ReactNode} from 'react';
import clsx from 'clsx';

import styles from './Sidebar.module.css';

export interface ISidebarProps {
  classNames?: {
    wrap?: string;
    content?: string;
    expand?: string;
    expandActive?: string;
  };
  children: ReactNode;
  renderExpand?: ReactNode;
  isExpanded?: boolean;
}

const Sidebar: FunctionComponent<ISidebarProps> = ({
  children,
  renderExpand,
  isExpanded,
  classNames = {},
}) => {
  const {content, expand, wrap, expandActive} = classNames;
  return (
    <div className={clsx(styles.wrap, wrap)}>
      <div className={clsx(styles.content, content)}>{children}</div>
      <div
        className={clsx(
          styles.expand,
          expand,
          isExpanded && styles.expandActive,
          isExpanded && expandActive,
        )}
      >
        {renderExpand}
      </div>
    </div>
  );
};

export default Sidebar;
