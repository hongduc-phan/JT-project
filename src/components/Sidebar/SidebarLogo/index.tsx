export * from './SidebarLogo';
export {default} from './SidebarLogo';
