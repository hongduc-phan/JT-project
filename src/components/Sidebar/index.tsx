export {default} from './Sidebar';
export {default as SidebarButton} from './SidebarButton';
export {default as SidebarMenu} from './SidebarMenu';
export {default as SidebarTitle} from './SidebarTitle';
export {default as SidebarMenuItem} from './SidebarMenuItem';
export {default as SidebarLogo} from './SidebarLogo';
