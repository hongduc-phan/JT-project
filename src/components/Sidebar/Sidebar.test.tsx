import React from 'react';
import renderer from 'react-test-renderer';

import Sidebar, {
  SidebarButton,
  SidebarMenu,
  SidebarMenuItem,
  SidebarTitle,
} from './';

import {ETime, EPayRoll, Setting} from '../Icons';

import {SidebarMenuVariant} from './types';

describe('<Sidebar />', () => {
  describe('Normal', () => {
    const SidebarNormalContainer = (isExpanded: boolean) => (
      <Sidebar
        isExpanded={isExpanded}
        classNames={{
          content: 'testContent',
          expand: 'testExpand',
          wrap: 'testWrap',
        }}
        renderExpand={
          <div>
            <SidebarTitle>Settings</SidebarTitle>
            <SidebarMenu>
              <SidebarMenuItem
                active={true}
                sub={
                  <SidebarMenu variant={SidebarMenuVariant.Sub}>
                    <SidebarMenuItem active={true}>Company</SidebarMenuItem>
                    <SidebarMenuItem>Departments</SidebarMenuItem>
                    <SidebarMenuItem>Gradings</SidebarMenuItem>
                    <SidebarMenuItem>Branches</SidebarMenuItem>
                    <SidebarMenuItem>WorkFlows</SidebarMenuItem>
                  </SidebarMenu>
                }
              >
                General
              </SidebarMenuItem>
              <SidebarMenuItem sub={<div />}>eTime</SidebarMenuItem>
              <SidebarMenuItem sub={<div />}>ePayroll</SidebarMenuItem>
              <SidebarMenuItem>Users</SidebarMenuItem>
            </SidebarMenu>
          </div>
        }
      >
        <SidebarButton>
          <ETime />
        </SidebarButton>
        <SidebarButton>
          <EPayRoll />
        </SidebarButton>
        <SidebarButton active={true}>
          <Setting active={true} />
        </SidebarButton>
      </Sidebar>
    );

    it('Should match normal sidebar snapshot', () => {
      const SidebarNormal = SidebarNormalContainer(false);

      const tree = renderer.create(SidebarNormal).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it('Should match sidebar expanded snapshot', () => {
      const SidebarNormal = SidebarNormalContainer(true);

      const tree = renderer.create(SidebarNormal).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
