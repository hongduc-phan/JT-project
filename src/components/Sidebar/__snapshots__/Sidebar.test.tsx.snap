// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> Normal Should match normal sidebar snapshot 1`] = `
<div
  className="wrap testWrap"
>
  <div
    className="content testContent"
  >
    <button
      className="icon"
    >
      <svg
        height="1em"
        viewBox="0 0 24 24"
        width="1em"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0h24v24H0z"
          />
          <path
            d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8zm.5-13H11v6l5.25 3.15.75-1.23-4.5-2.67V7z"
            fill="currentColor"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </button>
    <button
      className="icon"
    >
      <svg
        height="1em"
        viewBox="0 0 24 24"
        width="1em"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 3v13.714h24V3H0zm8.598 12H4.291a2.574 2.574 0 0 0-2.577-2.577v-4.28a3.432 3.432 0 0 0 3.429-3.429h3.455c-1.066 1.26-1.74 3.097-1.74 5.143 0 2.047.674 3.884 1.74 5.143zm13.688-2.577A2.59 2.59 0 0 0 19.714 15h-4.312c1.066-1.259 1.74-3.096 1.74-5.143 0-2.046-.674-3.884-1.74-5.143h3.455a3.432 3.432 0 0 0 3.429 3.429v4.28zM0 18.43h24v2H0v-2z"
            fill="currentColor"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </button>
    <button
      className="icon iconActive"
    >
      <svg
        height="1em"
        viewBox="0 0 24 24"
        width="1em"
      >
        <path
          d="M21.07 13.2c.06-.36.06-.78.06-1.2 0-.42-.06-.78-.06-1.2l2.564-1.98c.245-.18.306-.48.123-.78L21.314 3.9c-.122-.24-.489-.36-.733-.24l-3.054 1.2c-.61-.48-1.343-.9-2.076-1.2L15.023.48a.655.655 0 0 0-.61-.48H9.525c-.305 0-.55.24-.61.48l-.489 3.18c-.733.3-1.404.72-2.077 1.2l-3.053-1.2c-.306-.12-.611 0-.733.24L.12 8.04c-.122.24-.061.6.122.78l2.626 1.98c0 .42-.06.78-.06 1.2 0 .42.06.78.06 1.2L.304 15.18c-.244.18-.305.48-.122.78l2.443 4.14c.122.24.489.36.733.24l3.054-1.2c.61.48 1.343.9 2.076 1.2l.489 3.18c.061.3.305.48.61.48h4.886c.306 0 .55-.24.611-.48l.49-3.18c.732-.3 1.404-.72 2.075-1.2l3.054 1.2c.306.12.611 0 .733-.24l2.443-4.14c.122-.24.061-.6-.122-.78L21.07 13.2zm-9.1 3c-2.383 0-4.276-1.86-4.276-4.2s1.893-4.2 4.275-4.2c2.383 0 4.276 1.86 4.276 4.2s-1.893 4.2-4.276 4.2z"
          fill="currentColor"
          fillRule="nonzero"
        />
        <path
          d="M11.97 16.2c-2.383 0-4.276-1.86-4.276-4.2s1.893-4.2 4.275-4.2c2.383 0 4.276 1.86 4.276 4.2s-1.893 4.2-4.276 4.2z"
          fill="#5AC46A"
        />
      </svg>
    </button>
  </div>
  <div
    className="expand testExpand"
  >
    <div>
      <div
        className="typo title subTitle black left"
      >
        Settings
      </div>
      <ul
        className="menus"
      >
        <div
          className="Collapsible"
        >
          <span
            className="Collapsible__trigger is-open"
            onClick={[Function]}
            onKeyPress={[Function]}
            style={null}
            tabIndex={null}
          >
            <li
              className="menu menuActive"
            >
              <span
                className="typo h5 inherit left"
              >
                General
              </span>
              <span
                className="menuIcon"
              >
                <svg
                  height="1em"
                  viewBox="0 0 20 20"
                  width="1em"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                  >
                    <path
                      d="M6.175 13.333L10 9.517l3.825 3.816L15 12.158l-5-5-5 5z"
                      fill="currentColor"
                      fillRule="nonzero"
                    />
                    <path
                      d="M0 0h20v20H0z"
                    />
                  </g>
                </svg>
              </span>
            </li>
          </span>
          <div
            className="Collapsible__contentOuter"
            onTransitionEnd={[Function]}
            style={
              Object {
                "WebkitTransition": "none",
                "height": "auto",
                "msTransition": "none",
                "overflow": "hidden",
                "transition": "none",
              }
            }
          >
            <div
              className="Collapsible__contentInner"
            >
              <ul
                className="menus"
              >
                <li
                  className="subMenu subMenuActive"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Company
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Departments
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Gradings
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Branches
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    WorkFlows
                  </span>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <div
          className="Collapsible"
        >
          <span
            className="Collapsible__trigger is-closed"
            onClick={[Function]}
            onKeyPress={[Function]}
            style={null}
            tabIndex={null}
          >
            <li
              className="menu"
            >
              <span
                className="typo h5 inherit left"
              >
                eTime
              </span>
              <span
                className="menuIcon"
              >
                <svg
                  height="1em"
                  viewBox="0 0 20 20"
                  width="1em"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                  >
                    <path
                      d="M6.175 6.667L10 10.483l3.825-3.816L15 7.842l-5 5-5-5z"
                      fill="currentColor"
                      fillRule="nonzero"
                    />
                    <path
                      d="M0 20h20V0H0z"
                    />
                  </g>
                </svg>
              </span>
            </li>
          </span>
          <div
            className="Collapsible__contentOuter"
            onTransitionEnd={[Function]}
            style={
              Object {
                "WebkitTransition": "height 400ms ease-in-out",
                "height": 0,
                "msTransition": "height 400ms ease-in-out",
                "overflow": "hidden",
                "transition": "height 400ms ease-in-out",
              }
            }
          >
            <div
              className="Collapsible__contentInner"
            >
              <div />
            </div>
          </div>
        </div>
        <div
          className="Collapsible"
        >
          <span
            className="Collapsible__trigger is-closed"
            onClick={[Function]}
            onKeyPress={[Function]}
            style={null}
            tabIndex={null}
          >
            <li
              className="menu"
            >
              <span
                className="typo h5 inherit left"
              >
                ePayroll
              </span>
              <span
                className="menuIcon"
              >
                <svg
                  height="1em"
                  viewBox="0 0 20 20"
                  width="1em"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                  >
                    <path
                      d="M6.175 6.667L10 10.483l3.825-3.816L15 7.842l-5 5-5-5z"
                      fill="currentColor"
                      fillRule="nonzero"
                    />
                    <path
                      d="M0 20h20V0H0z"
                    />
                  </g>
                </svg>
              </span>
            </li>
          </span>
          <div
            className="Collapsible__contentOuter"
            onTransitionEnd={[Function]}
            style={
              Object {
                "WebkitTransition": "height 400ms ease-in-out",
                "height": 0,
                "msTransition": "height 400ms ease-in-out",
                "overflow": "hidden",
                "transition": "height 400ms ease-in-out",
              }
            }
          >
            <div
              className="Collapsible__contentInner"
            >
              <div />
            </div>
          </div>
        </div>
        <li
          className="menu"
        >
          <span
            className="typo h5 inherit left"
          >
            Users
          </span>
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`<Sidebar /> Normal Should match sidebar expanded snapshot 1`] = `
<div
  className="wrap testWrap"
>
  <div
    className="content testContent"
  >
    <button
      className="icon"
    >
      <svg
        height="1em"
        viewBox="0 0 24 24"
        width="1em"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0h24v24H0z"
          />
          <path
            d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8zm.5-13H11v6l5.25 3.15.75-1.23-4.5-2.67V7z"
            fill="currentColor"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </button>
    <button
      className="icon"
    >
      <svg
        height="1em"
        viewBox="0 0 24 24"
        width="1em"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 3v13.714h24V3H0zm8.598 12H4.291a2.574 2.574 0 0 0-2.577-2.577v-4.28a3.432 3.432 0 0 0 3.429-3.429h3.455c-1.066 1.26-1.74 3.097-1.74 5.143 0 2.047.674 3.884 1.74 5.143zm13.688-2.577A2.59 2.59 0 0 0 19.714 15h-4.312c1.066-1.259 1.74-3.096 1.74-5.143 0-2.046-.674-3.884-1.74-5.143h3.455a3.432 3.432 0 0 0 3.429 3.429v4.28zM0 18.43h24v2H0v-2z"
            fill="currentColor"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </button>
    <button
      className="icon iconActive"
    >
      <svg
        height="1em"
        viewBox="0 0 24 24"
        width="1em"
      >
        <path
          d="M21.07 13.2c.06-.36.06-.78.06-1.2 0-.42-.06-.78-.06-1.2l2.564-1.98c.245-.18.306-.48.123-.78L21.314 3.9c-.122-.24-.489-.36-.733-.24l-3.054 1.2c-.61-.48-1.343-.9-2.076-1.2L15.023.48a.655.655 0 0 0-.61-.48H9.525c-.305 0-.55.24-.61.48l-.489 3.18c-.733.3-1.404.72-2.077 1.2l-3.053-1.2c-.306-.12-.611 0-.733.24L.12 8.04c-.122.24-.061.6.122.78l2.626 1.98c0 .42-.06.78-.06 1.2 0 .42.06.78.06 1.2L.304 15.18c-.244.18-.305.48-.122.78l2.443 4.14c.122.24.489.36.733.24l3.054-1.2c.61.48 1.343.9 2.076 1.2l.489 3.18c.061.3.305.48.61.48h4.886c.306 0 .55-.24.611-.48l.49-3.18c.732-.3 1.404-.72 2.075-1.2l3.054 1.2c.306.12.611 0 .733-.24l2.443-4.14c.122-.24.061-.6-.122-.78L21.07 13.2zm-9.1 3c-2.383 0-4.276-1.86-4.276-4.2s1.893-4.2 4.275-4.2c2.383 0 4.276 1.86 4.276 4.2s-1.893 4.2-4.276 4.2z"
          fill="currentColor"
          fillRule="nonzero"
        />
        <path
          d="M11.97 16.2c-2.383 0-4.276-1.86-4.276-4.2s1.893-4.2 4.275-4.2c2.383 0 4.276 1.86 4.276 4.2s-1.893 4.2-4.276 4.2z"
          fill="#5AC46A"
        />
      </svg>
    </button>
  </div>
  <div
    className="expand testExpand expandActive"
  >
    <div>
      <div
        className="typo title subTitle black left"
      >
        Settings
      </div>
      <ul
        className="menus"
      >
        <div
          className="Collapsible"
        >
          <span
            className="Collapsible__trigger is-open"
            onClick={[Function]}
            onKeyPress={[Function]}
            style={null}
            tabIndex={null}
          >
            <li
              className="menu menuActive"
            >
              <span
                className="typo h5 inherit left"
              >
                General
              </span>
              <span
                className="menuIcon"
              >
                <svg
                  height="1em"
                  viewBox="0 0 20 20"
                  width="1em"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                  >
                    <path
                      d="M6.175 13.333L10 9.517l3.825 3.816L15 12.158l-5-5-5 5z"
                      fill="currentColor"
                      fillRule="nonzero"
                    />
                    <path
                      d="M0 0h20v20H0z"
                    />
                  </g>
                </svg>
              </span>
            </li>
          </span>
          <div
            className="Collapsible__contentOuter"
            onTransitionEnd={[Function]}
            style={
              Object {
                "WebkitTransition": "none",
                "height": "auto",
                "msTransition": "none",
                "overflow": "hidden",
                "transition": "none",
              }
            }
          >
            <div
              className="Collapsible__contentInner"
            >
              <ul
                className="menus"
              >
                <li
                  className="subMenu subMenuActive"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Company
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Departments
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Gradings
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    Branches
                  </span>
                </li>
                <li
                  className="subMenu"
                >
                  <span
                    className="typo h5 inherit left"
                  >
                    WorkFlows
                  </span>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <div
          className="Collapsible"
        >
          <span
            className="Collapsible__trigger is-closed"
            onClick={[Function]}
            onKeyPress={[Function]}
            style={null}
            tabIndex={null}
          >
            <li
              className="menu"
            >
              <span
                className="typo h5 inherit left"
              >
                eTime
              </span>
              <span
                className="menuIcon"
              >
                <svg
                  height="1em"
                  viewBox="0 0 20 20"
                  width="1em"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                  >
                    <path
                      d="M6.175 6.667L10 10.483l3.825-3.816L15 7.842l-5 5-5-5z"
                      fill="currentColor"
                      fillRule="nonzero"
                    />
                    <path
                      d="M0 20h20V0H0z"
                    />
                  </g>
                </svg>
              </span>
            </li>
          </span>
          <div
            className="Collapsible__contentOuter"
            onTransitionEnd={[Function]}
            style={
              Object {
                "WebkitTransition": "height 400ms ease-in-out",
                "height": 0,
                "msTransition": "height 400ms ease-in-out",
                "overflow": "hidden",
                "transition": "height 400ms ease-in-out",
              }
            }
          >
            <div
              className="Collapsible__contentInner"
            >
              <div />
            </div>
          </div>
        </div>
        <div
          className="Collapsible"
        >
          <span
            className="Collapsible__trigger is-closed"
            onClick={[Function]}
            onKeyPress={[Function]}
            style={null}
            tabIndex={null}
          >
            <li
              className="menu"
            >
              <span
                className="typo h5 inherit left"
              >
                ePayroll
              </span>
              <span
                className="menuIcon"
              >
                <svg
                  height="1em"
                  viewBox="0 0 20 20"
                  width="1em"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                  >
                    <path
                      d="M6.175 6.667L10 10.483l3.825-3.816L15 7.842l-5 5-5-5z"
                      fill="currentColor"
                      fillRule="nonzero"
                    />
                    <path
                      d="M0 20h20V0H0z"
                    />
                  </g>
                </svg>
              </span>
            </li>
          </span>
          <div
            className="Collapsible__contentOuter"
            onTransitionEnd={[Function]}
            style={
              Object {
                "WebkitTransition": "height 400ms ease-in-out",
                "height": 0,
                "msTransition": "height 400ms ease-in-out",
                "overflow": "hidden",
                "transition": "height 400ms ease-in-out",
              }
            }
          >
            <div
              className="Collapsible__contentInner"
            >
              <div />
            </div>
          </div>
        </div>
        <li
          className="menu"
        >
          <span
            className="typo h5 inherit left"
          >
            Users
          </span>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
